faunadb/request_response_info.py

```diff
@@ -0,0 +1,45 @@
+class RequestResponseInfo(object):
```

Comment:
We're not really passing requests and responses around. Those all stay within `Client`. Did you think there's a good place to use these classes in there?